tests/30rooms/06invite.pl

```diff
@@ -107,3 +111,424 @@
          Future->done(1);
       });
    };
+
```

Comment:
There's a lot of new tests here and they're all related to 3rdparty invites. Maybe they want to be in their own file?